### PR TITLE
added i2c table and led tables and pictures

### PR DESCRIPTION
**Pull request by jason-jaia**

### CODE REVIEW

File: src/doc/figures/power-board-leds.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (ferrom-jaia) [inline src/doc/figures/power-board-leds.png:Zone.Identifier]: Do we need this file?

File: src/doc/figures/logic-board-leds.png:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (ferrom-jaia) [inline src/doc/figures/logic-board-leds.png:Zone.Identifier]: Do we need this file?

Comment (jason-jaia) [reply src/doc/figures/logic-board-leds.png:Zone.Identifier]: No - I have no idea what it is.

**Review by ferrom-jaia — APPROVED** [review]
Sweet! Reviewed.

**Pull request merged**